emoji-ui/src/components/EmojiList.css

```diff
@@ -1,4 +1,4 @@
 .emoji-list-container {
-  width: 70%;
+  width: 800px;
```

Comment:
i liked the resizeable, mobile-friendly ui though :p

Reply:
how about just give it a 40px margin/padding and leave it at the default 100% width?

Reply:
i add specific width so i can place 120 emojis and have it filled the last row except the last page. it is a purely cosmetic thing just for looking pretty for demo. let's go with this hack for now 👍